controller_interface/CMakeLists.txt

```diff
@@ -27,6 +27,8 @@ ament_target_dependencies(
 # Causes the visibility macros to use dllexport rather than dllimport,
 # which is appropriate when building the dll but not consuming it.
 target_compile_definitions(controller_interface PRIVATE "CONTROLLER_INTERFACE_BUILDING_DLL")
+# prevent pluginlib from using boost
+target_compile_definitions(controller_interface PUBLIC "PLUGINLIB__DISABLE_BOOST_FUNCTIONS")
```

Comment:
@Karsten1987 hello dear Sir, do you think it'd be possible to flip this around and make people *enable* boost functionality if they need it, given that we are so strongly encouraged to not use boost? Or is this already done somewhere upper-stream?

Reply:
We'll have it in Galactic: https://github.com/ros/pluginlib/pull/199